lib/serializer.ts

```diff
@@ -97,10 +97,10 @@ class ResolvedDeclarationEmitter {
       // The declaration node may not be a complete statement, e.g. for var/const
       // symbols. We need to find the complete export statement by traversing
       // upwards.
-      while (!(decl.flags & ts.NodeFlags.Export) && decl.parent) {
```

Comment:
Some of the flags were extracted into `ModifierFlags` and are not available from `Node.flags` anymore. `getCombinedModifierFlags` should be used to access them, but it collects flags from all descendant nodes, so it is not a replacement for `Node.flags`. I had to use private API to access only flags from the node at hand.